### ISSUE TITLE
Problems with column paste at end of file

### ISSUE THREAD

**Issue by darnn** [SubtitleEdit/subtitleedit#3181]
3.5.7 build 102.
Caused by the most recent changes to column paste, I assume.
Copy two subtitles to the clipboard. Create a new file and in the file create two subtitles. Try to perform column paste. The text will not be pasted. Now add a third subtitle to the file and try to use column paste on the first subtitle. It will work properly. If you try on the second subtitle, it will fail to paste once again.
So, in short, if you try to column paste two subtitles into the last two subtitles of a file, it won't work.
In fact, now that I look at it, even trying to column paste a single subtitle into one of the last two subtitles of a file doesn't work.
But you can, for instance, column paste three subtitles into the last three subtitles in a file.

**Comment by OmrSi** [comment]
I was literally just coming here to report this, I had two identical lines at the beginning and end of the file, when I tried to paste it, it didn't work...

**Comment by niksedk** [comment]
Nice testing :)
How is this: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.7/SubtitleEditBeta.zip

**Comment by darnn** [comment]
Seems to work over here! Thanks!
